swagger_server/messaging/topic_queue_consumer.py

```diff
@@ -90,8 +90,14 @@ def handle_mq_msg(self, msg_body):
                 self.message_id += 1
                 self.logger.info("Sending connection info to Kytos.")
                 # Uncomment lines below to send connection info to Kytos
-                r = requests.post(str(KYTOS_URL), json=msg_json)
-                self.logger.info("Status code:" + str(r.status_code))
+                try:
+                    r = requests.post(str(KYTOS_URL), json=msg_json)
+                    self.logger.info("Status code:" + str(r.status_code))
```

Comment:
We could use an f-string here:

```python
self.logger.info(f"Status code from Kytos {r.status_code}")
```

Or maybe the below, if `f"{r}"` it is not a huge string:

```python
self.logger.info(f"Status from Kytos {r}")
```